Fixing substore notation in module LIPs (#234)
LiskHQ/lips: proposals/lip-0047.md

* The substore prefix is set to `SUBSTORE_PREFIX_NAME`.
* Each substore key is a name of a validator as given in the genesis block or with an [register authority command](#register-authority-command), serialized as a utf-8 encoded string.
* Each substore value is set to the address of the corresponding validator, serialized according to the JSON schema `validatorAddressSchema` below.

> * Notation: let `name(nameBytes)` denote the entry in the name substore with the key `nameBytes`, deserialized using the `validatorAddressSchema` schema..

Reviewer: ```suggestion
* Notation: let `name(nameBytes)` denote the entry in the name substore with the key `nameBytes`, deserialized using the `validatorAddressSchema` schema.
```

Author: Done

* The substore prefix is set to `SUBSTORE_PREFIX_SNAPSHOT`.
* Each substore key is `uint32be(roundNumber)`, where `roundNumber` can be 0, 1 or 2 corresponding to the current round, the next round and in two rounds respectively.
* Each substore value is the serialization of an object following `snapshotStoreSchema`.

> * Notation: Let `snapshotStore(roundNumber)` be the substore entry with prefix `SUBSTORE_PREFIX_SNAPSHOT`, deserialized using the `snapshotStoreSchema` schema,  and key `uint32be(roundNumber)`.

Reviewer: To be consistent with `Notation` part for other substores:
```suggestion
* Notation: Let `snapshotStore(roundNumber)` be the substore entry in the snapshot substore with the key `uint32be(roundNumber)`, deserialized using the `snapshotStoreSchema` schema.
```

Author: Done
